frontend/apps/reader/modules/readerdictionary.lua

```diff
@@ -273,15 +274,26 @@ function ReaderDictionary:addToMainMenu(menu_items)
                 sub_item_table_func = function() return self:_genDownloadDictionariesMenu() end,
             },
             {
-                text = _("Enable fuzzy search"),
+                text_func = function()
+                    local text = _("Enable fuzzy search")
+                    if G_reader_settings:nilOrFalse("disable_fuzzy_search") then
+                        text = text .. "  ★"
+                    end
+                    return text
+                end,
                 checked_func = function()
-                    return self.disable_fuzzy_search ~= true
+                    return not (self.ui.doc_settings and self.disable_fuzzy_search or self.disable_fuzzy_search_fm)
```

Comment:
Doesn't that leak ? if `self.disable_fuzzy_search` is `false`, you would `use self.disable_fuzzy_search_fm` ?

Reply:
Sorry